index.html

```diff
@@ -0,0 +1,48 @@
+       GOLDEN BIRD </h2>
+    <nav id = "nav-bar">

```

Comment:
please don't use space between property and property-value like `id = "nav-bab"` it would be better to write it like `id="nav-bar"`.